src/main/java/org/sagebionetworks/warehouse/workers/db/snapshot/AccessRecordDaoImpl.java

```diff
@@ -175,6 +178,7 @@ public void setValues(PreparedStatement ps, int i)
 						ps.setBoolean(19, ar.getSuccess());
 						ps.setLong(20, ar.getResponseStatus());
 						ps.setString(21, ar.getOauthClientId());
+						ps.setString(22, ar.getBasicAuthUsername());
```

Comment:
We might want to normalize this before storage? (e.g. trim and lowercase) Or alternatively we can store it in the normalized access record normalized.